docs/modules/configure/pages/initializer.adoc

```diff
@@ -441,3 +441,72 @@ To customize export fields, you can subscribe to any serialize event. Every seri
   end
 ....
 
+== Content Security Policy
+Defines additional content security policies following the structure keys are the CSP directives and the values are arrays of allowed sources
+
+List of additional content security policies to be appended to the default ones
+This is useful for adding custom CSPs for external services like Here Maps, YouTube, Twitter etc.
+
+Below you will find a the recepy of enabling Twitter timelines, based on https://content-security-policy.com/examples/twitter/ example.
+The format is a Hash with the following structure:
+[source,ruby]
+....
+  {
+    "script-src": %w(platform.twitter.com syndication.twitter.com),
+    "style-src": %w('sha256-5g0QXxO6NfvHJ6Uf5BK/hqQHtso8ZOdjlnbyKtYLvwc='),
+    "frame-src": %w(platform.twitter.com),
+    "img-src": %w(syndication.twitter.com),
+  }
+....
+
+Additionally, the following example, will allow you to enable Jitsi meetings:
+
+[source,ruby]
+....
+  {
+    "connect-src": %w(meet.jit.si),
+    "frame-src": %w(meet.jit.si),
+  }
+....
```

Comment:
I'd remove this example, as the typical use-case for jitsi meeting should be covered with the online meetings integration that we already have. Maybe we can find other popular external platform that isn't yet covered (like youtube)?